mobidam-sst-management-backend/src/main/java/de/muenchen/mobidam/domain/Schnittstelle.java

```diff
@@ -56,7 +56,6 @@ public class Schnittstelle extends BaseEntity {
 
     @Column(nullable = false)
     @NotNull
-    @FutureOrPresent
```

Comment:
Das ist ja so fachl. nicht richtig. Das Anlagedatum sollte immer in der Zukunft liegen und bei einem Update auch nicht überschrieben werden. Ggf. kannst Du das Datum aus dem Dto nehmen.

Reply:
ja, leider ist das nicht so einfach. Die Annotation schmeißt einen Fehler egal ob da was gesetzt ist oder nicht

Reply:
Ach ja, beim Überschreiben der Schnittstelle ist das Datum ja immer in der Vergangenheit, d.h. die Annotation stimmt nicht.